src/qt/locale/bitcoin_fr.ts

```diff
@@ -4915,9 +4929,13 @@ Impossible de restaurer la sauvegarde du porte-monnaie</translation>
         <source>The transaction amount is too small to pay the fee</source>
         <translation type="unfinished">Le montant de la transaction est trop bas pour que les frais soient payés</translation>
     </message>
+    <message>
+        <source>The transactions removal process can only be executed within a db txn</source>
+        <translation type="unfinished">Le processus de suppression des transactions ne peut être exécuté que dans une transaction .de base de données</translation>
```

Comment:
Seems harmless but there is some spurious symbol inserted in some sentences, such as this period here.